litmus-portal/frontend/src/components/WelcomeModal/Stepper.tsx

```diff
@@ -0,0 +1,294 @@
+                data-cy="inputProjectEmail"
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setData('email', event.target.value);
+                }}
+              />
+              </ Paper>
+            }
+            //pass here corresponding name of user
+            setName={"Name"}
+            setText={'To get started, enter the email *optional*'}
+          />
+        );
+      default:
+        return (
+          <ModalPage
+            renderMenu={
+              <Paper className={classes.inputArea}>  
+              <TextField
+                id="filled-projectname-input"
+                label="Project Name"
+                InputProps={{ disableUnderline: true }}
+                data-cy="inputProjectEmail"
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setData('nameProject', event.target.value);
+                }}
+              />
+              </ Paper>
+            }
+            setName={"Administrator"}
+            setText={'To get started, enter the name of your first project.'}
+          />
+        );
+    }
+  };
+
+  /* Reset is used to reset the steps and further can be used */
+  /* to route but keep handlereset */
+  return (
+    <div>
+      <div>
+        {activeStep === steps.length-1? (
+          <div>
+            <Typography>{getStepContent(activeStep)}</Typography>
+            
+            <div>
+            <Button className={classes.buttonOutline} disabled={activeStep === 0} onClick={handleBack}>
+                Back
+              </Button>
+              <Button
+                variant="contained"
+                color="secondary"
+                onClick={() => {
+                  updateData();
+
+                  // close the Modal please look into it 
+                  funck();
+                  // add route here for the home page direction
+                }}
+                data-cy="selectProjectFinish"
+              >
+                Let&#39;s start
+              </Button>
+            </div>
+          </div>
+        ) : (
+          <div>
+            <Typography>{getStepContent(activeStep)}</Typography>
+            <div>
+              <Button className={classes.buttonOutline} disabled={activeStep === 0} onClick={handleBack}>
+                Back
+              </Button>
+              <Button
+                variant="contained"
+                color="secondary"
+                onClick={handleNext}
+                data-cy="Welcome-continue"
+              >
+                {activeStep === steps.length ? "Let's start" : 'Continue'}
+              </Button>
+            </div>
+          </div>
+        )}
+      </div>
+      <div className={classes.stepper}>
+        <Stepper
+          alternativeLabel
+          activeStep={activeStep}
+          data-cy="Welcome-stepper"
+        >
+          {steps.map((label) => (
```

Comment:
Again the values of steps are "" so the individual labels are also "" which makes all the keys same too. No point in having this logic

Reply:
What should i use here?